README.md

```diff
@@ -1 +1,36 @@
-# ProjectReview__Rep
\ No newline at end of file
+
+## ProjectReview__Rep
+> This is a trial htmls and css file done in collaboration.
```

Comment:
please write a descriptive paragraph that can describe the project for users. because this is where the user will come to get a glance at what the project is all about.